test/e2e/framework/node/resource.go

```diff
@@ -217,36 +217,26 @@ func TotalReady(c clientset.Interface) (int, error) {
 	return len(nodes.Items), nil
 }
 
-// GetExternalIP returns node external IP concatenated with port 22 for ssh
+// GetExternalSSHIP returns node external IP concatenated with port 22 for ssh
 // e.g. 1.2.3.4:22
-func GetExternalIP(node *v1.Node) (string, error) {
+func GetExternalSSHIP(node *v1.Node) (string, error) {
```

Comment:
can we avoid changing names to these exported functions in this PR?

I'd like to avoid touching other tests 😄

Reply:
Yes, reverted :) Can do a separate PR if you think that's make sense, or we can leave as it is.

Reply:
separate PR better, I'd like to backport it and make it as much targeted as possible

Reply:
sure, updated!